documentation/assemblies/assembly-kafka-connect-external-configuration.adoc

```diff
@@ -11,14 +11,13 @@
 
 = Using external configuration and secrets
 
-Kafka Connect connectors are configured using an HTTP REST interface.
-The connector configuration is passed to Kafka Connect as part of an HTTP request and stored within Kafka itself.
+Connectors are created, reconfigured, and deleted using an HTTP REST interface unless `KafkaConnectors` are enabled for the Kafka Connect cluster (for more information, see xref:con-creating-managing-connectors-{context}[]). The connector configuration is passed to Kafka Connect as part of an HTTP request and stored within Kafka itself.
 
-Some parts of the configuration of a Kafka Connect connector can be externalized using ConfigMaps or Secrets.
+Some parts of the configuration of a connector can be externalized using ConfigMaps or Secrets.
```

Comment:
I might move up the sentence _ConfigMaps and Secrets are standard Kubernetes resources used for storing configurations and confidential data._ , and then say, _You can use ConfigMaps and Secrets to configure certain aspects of a connector._ 

_Externalize_ has a different meaning for me, like convey or something. 
What can't we configure externally?